src/amo/components/Addon/styles.scss

```diff
@@ -109,46 +127,22 @@
   width: 100%;
 }
 
-.Addon-title {
-  order: 1;
-
+.Addon-summary-and-install-button-wrapper {
   @include respond-to(medium) {
-    flex-grow: 1;
-  }
-}
-
-.Addon .AddonMeta {
-  order: 2;
-
-  @include respond-to(medium) {
-    @include margin-start(auto);
-
-    margin: 10px 0;
-    order: 0;
-    text-align: right;
-  }
-}
-
-.Addon-persona .AddonMeta {
-  @include respond-to(medium) {
-    order: 1;
-  }
-}
-
-.Addon .AddonMeta-item {
-  @include margin-start(auto);
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  };
```

Comment:
Linter complains because of this extra semi-colon

Reply:
Oops, good catch, thanks.